src/pages/ContentScripts/features/repo-header-labels/view.tsx

```diff
@@ -37,6 +37,27 @@ const View = ({ activity, openrank, participant, contributor, meta }: Props): JS
   const participantData = generateDataByMonth(participant, meta.updatedAt);
   const contributorData = generateDataByMonth(contributor, meta.updatedAt);
 
+  type Label = {
+    id: string;
+    name: string;
+    type: string;
+  };
+
+  const labels: Label[] = meta.labels as Label[];
+
+  const spans = labels.map((label, index) => (
```

Comment:
We need a better namig instead of spans, maybe use the feature name `meta_labels`. I am not familiar with the names of the data. @wangyantong2000

Reply:
They are label data of repo. Maybe it can be called `repo_labels`.

<img width="558" alt="image" src="https://github.com/user-attachments/assets/1c6197ba-2c71-4b35-af36-706d4cdc479e">

Reply:
> We need a better namig instead of spans, maybe use the feature name `meta_labels`. I am not familiar with the names of the data. @wangyantong2000

They are label data of repo. Maybe it can be called `repo_labels`.

<img width="558" alt="image" src="https://github.com/user-attachments/assets/1c6197ba-2c71-4b35-af36-706d4cdc479e">